### ISSUE TITLE
Store signed in users games?

### ISSUE THREAD

**Issue by johndrinkwater** [SteamLUG/steamlug.org#131]
To highlight events and servers that would apply for their library, we could do this.
Thoughts?

**Comment by Cheeseness** [comment]
Yeah, that could be neat. With the amount of time it takes for the group check to happen, we should probably make sure we're being mindful of processing time, though.

**Comment by johndrinkwater** [comment]
Yes, i’d noticed the time to log in was a little slower with the avatar + personaname added, but there is little we can do beyond looking at threading or doing that work at the end of the page run. We need that information by the time the user starts using the site, or they’ll not understand why it does not work.
I have a reasonable library size; if the testing I do when I get around to adding this makes the process too slow, we shall postpone it.

**Comment by Cheeseness** [comment]
Yeah, I'm not sure what the best direction for this in general is. Maybe caching this between logins is worthwhile, but it's kinda nice to only be storing this stuff during the session so that we can be assured that it's fresh every time.
